src/Components/quiz/play/Quiz.jsx

```diff
@@ -17,9 +29,11 @@ what Quiz does:
   - create changeArray to record investment in firestore
 */
  
-// TODO: get this data from firestore
-const NUM_SYMBOLS = 3;
-const BALANCE = 1000;
+// TODO: get this data from parent
```

Comment:
This can potentially be handled in a later PR, but I'm going to mention it here. The current setup you have it right now is (from what I understand comment if I'm wrong.) you incrementally log their Buy hold or sells, and then when they click send. You parse it and send it. One issue with that approach is that if a person wants to sell one of their stocks to buy another one(something very common in the stock market) They're unable to do it. Instead, I think instead of saving at each step make it so they can incrementally buy and hold stocks(you can do this with states) and have it increment. Then if they don't do anything assume they've held a stock. This way in rounds with a large number of stocks a user doesn't have to input every single option. They also are able to sell stocks to buy bigger ones.  This can be handled in a later commit but I'm bringing it up now.

Reply:
thats a good point. I think for the sake of the prototype I'll leave it as-is, but once we get that working ill definitely add this. thanks!